src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -50,12 +54,51 @@ const VideoDetailPage = () => {
     features.FEATURE_ENABLE_VIDEO_CATALOG
     && hasActivatedAndCurrentSubscription(subscriptionLicense)
   );
+  useEffect(() => {
+    sendEnterpriseTrackEvent(
+      enterpriseCustomer.uuid,
+      'edx.ui.enterprise.learner_portal.video.detail_page_viewed',
+      {
+        userId,
+        video: videoData?.videoUrl,
+        courseKey: videoData?.courseKey,
+        title: videoData?.courseTitle,
+      },
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Shouldn't we add `videoData?.videoURL` as a dependency in the `useEffect`? I believe the `page_viewed` event should be triggered whenever the video URL changes. Additionally, I don't think a separate `useEffect` is necessary; we can integrate this logic into the existing `useEffect`.